lib/class-wp-rest-block-directory-controller.php

```diff
@@ -71,10 +71,12 @@ public function register_routes() {
 	 *
 	 * @since 6.5.0
 	 *
-	 * @param WP_REST_Request $request Full details about the request.
 	 * @return WP_Error|bool True if the request has permission, WP_Error object otherwise.
+	 *
+	 * This function is overloading a function defined in WP_REST_Controller so it should have the same parameters.
```

Comment:
Little confused by the comment as the `$request` param has been removed in the pr. 

I also couldn't see `permissions_check` defined in WP_REST_Controller. Instead there are individual methods (`get_items_permissions_check`, `get_item_permissions_check`, `create_item_permissions_check` ... ).

Reply:
Nice catch the disable is not needed here I removed it 👍